src/main/kotlin/uk/gov/justice/digital/hmpps/prisontonhs/services/RestResponsePage.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.prisontonhs.services
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.databind.JsonNode
+import org.springframework.data.domain.PageImpl
+import org.springframework.data.domain.PageRequest
+import org.springframework.data.domain.Pageable
+import java.util.*
+
+
+internal class RestResponsePage<T> : PageImpl<T> {
```

Comment:
This is done to allow de serialisation of page object

Reply:
If you were using Jackson you could have used `com.fasterxml.jackson.module:jackson-module-kotlin` to create constructors I think